index.html

```diff
@@ -1233,24 +1233,24 @@
 ## Data Minimization {#data-minimization}
 
 <div class="practice">
-<span class="practicelab">Sites, user agents, and other actors should minimize the amount of
-<a>personal data</a> they transfer between actors on the Web.</span>
+<span class="practicelab">[=Sites=], [=user agents=], and other [=actors=]
+should minimize the amount of [=personal data=] they transfer. [=User agents=]
+should share data only when needed, in alignment with a user's goals and
+interests. Web APIs should be designed to minimize the amount of data that
```

Comment:
This might lose an important nuance, but maybe:

```suggestion
should share data only when it helps a user's goals and
interests. Web APIs should be designed to minimize the amount of data that
```

Reply:
Simpler language, +1